test-templates/src/groups.rs

```diff
@@ -70,4 +74,14 @@ pub fn group_test<G: Group>(a: G, mut b: G) {
         a.mul(&(fr_rand1 * &fr_rand2)),
         "(a * r1) * r2 != a * (r1 * r2)"
     );
+
+    // Check that mul and wnaf_mul give the same results for several window values
+    for w in 2..=22 {
```

Comment:
Can we lower the max tested table size, I think building a 2^22 table is way too much computation for CI